<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainwindow</class>
 <widget class="QMainWindow" name="Mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualization</string>
  </property>
  <property name="windowIcon">
   <iconset resource="alg_vis.qrc">
    <normaloff>:/cgal/icons/resources/cgal_logo.xpm</normaloff>:/cgal/icons/resources/cgal_logo.xpm</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="Viewer" name="viewer" native="true">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoadPolyhedron"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionView_polyhedron"/>
   </widget>
   <widget class="QMenu" name="menuAlgorithms">
    <property name="title">
     <string>Algorithms</string>
    </property>
    <addaction name="actionSurface_simplification"/>
    <addaction name="separator"/>
    <addaction name="actionShape_detection"/>
    <addaction name="separator"/>
    <addaction name="actionHorizontal_plane_detection"/>
    <addaction name="separator"/>
    <addaction name="actionUnit_normal_detection"/>
    <addaction name="separator"/>
    <addaction name="actionSymmetric_normal_detection"/>
    <addaction name="separator"/>
    <addaction name="actionRidge_detection"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy_snapshot"/>
    <addaction name="actionSave_snapshot"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAlgorithms"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLoadPolyhedron">
   <property name="text">
    <string>Load polyhedron...</string>
   </property>
  </action>
  <action name="actionView_polyhedron">
   <property name="text">
    <string>Polyhedron</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionSave_snapshot">
   <property name="text">
    <string>Save snapshot</string>
   </property>
  </action>
  <action name="actionCopy_snapshot">
   <property name="text">
    <string>Copy snapshot</string>
   </property>
  </action>
  <action name="actionShape_detection">
   <property name="text">
    <string>Shape detection</string>
   </property>
  </action>
  <action name="actionSurface_simplification">
   <property name="text">
    <string>Surface simplification</string>
   </property>
  </action>
  <action name="actionRidge_detection">
   <property name="text">
    <string>Ridge detection</string>
   </property>
  </action>
  <action name="actionHorizontal_plane_detection">
   <property name="text">
    <string>Horizontal plane detection</string>
   </property>
  </action>
  <action name="actionUnit_normal_detection">
   <property name="text">
    <string>Unit normal detection</string>
   </property>
  </action>
  <action name="actionSymmetric_normal_detection">
   <property name="text">
    <string>Symmetric normal detection</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QWidget</extends>
   <header>Viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="alg_vis.qrc"/>
 </resources>
 <connections/>
</ui>
